### PR TITLE
Added instructions and support for have script policy provider workin…

### PR DESCRIPTION
**Pull request by mposolda**
…g on Java 17

### CODE REVIEW

**Comment by mposolda** [comment]
This is draft PR related to https://github.com/keycloak/keycloak/pull/11322 . See that PR for more details.

File: authz-js-policies/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+What is it?
+-----------
+
+TODO
+
+System Requirements
+-------------------
+
+You need to have Keycloak 18.0.Final running.
```
Comment (DGuhr) [inline authz-js-policies/README.md:19]: perhaps better "at least Keycloak 18.0.Final" or sth? I'd avoid strict version pinning in a general example.

File: authz-js-policies/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+All you need to build this project is Java 8.0 (Java SDK 1.8) or later and Maven 3.3.3 or later.
```
Comment (DGuhr) [inline authz-js-policies/README.md:21]: For Quarkus, it'd be Java11

File: authz-js-policies/README.md (hunk 1 above)
Comment (mposolda) [reply authz-js-policies/README.md]: Yes, true. Thanks for these points. I've copy/pasted this from the README of some other example :-) Similarly the Java version.

It seems that README files of the quickstarts, which are deployable to Keycloak server, may need some revision as they are currently target just for the Keycloak on Wildfly.

This particular quickstart didn't have README file like the other quickstarts have. So I've added README file, but for this README, I've added lots of TODO and I focused only on parts specific to instructions for deployment on Keycloak on Wildfly VS Keycloak on Quarkus as this is the most important part for the particular use-case addressed here for the scripts providers. I can properly update the README file and remove all TODOs if we agree that my approach for script providers work.

**Review by pedroigor — APPROVED** [review]

**Comment by mposolda** [comment]
@pedroigor @DGuhr Do you please have a chance to re-review this? Corresponding main codebase PR is also ready for the review: https://github.com/keycloak/keycloak/pull/11322 .
Thanks

**Review by pedroigor — APPROVED** [review]

**Pull request merged**